src/api/schema.graphql

```diff
@@ -889,6 +889,7 @@ input JointlyInsuredPartyCreateInput {
 type ExportContract {
   id: ID!
   application: Application
+  agent: ExportContractAgent
```

Comment:
Note for PR reviewers - this file is automatically generated by KeystoneJS (from updating schema.ts)